scripts/make-distribution

```diff
@@ -2,44 +2,83 @@
+
+  export PIPELINE_ADAPTER_IMG="syntasso.docker.scarf.sh/syntasso/kratix-platform-pipeline-adapter@${PIPELINE_ADAPTER_VERSION_MANIFEST_DIGEST}"
+  export IMG_TAG="syntasso.docker.scarf.sh/syntasso/kratix-platform@${KRATIX_PLATFORM_MANIFEST_DIGEST}"
+
+  make distribution
+  mkdir -p distribution/single-cluster
+
+  cat config/samples/platform_v1alpha1_worker.yaml <(echo "---") \
+      config/samples/platform_v1alpha1_bucketstatestore.yaml <(echo "---") \
+      hack/destination/gitops-tk-resources-single-cluster.yaml > distribution/single-cluster/config-all-in-one.yaml
+
+  cat distribution/kratix.yaml <(echo "---") \
+      hack/destination/gitops-tk-install.yaml <(echo "---") \
+      hack/platform/minio-install.yaml > distribution/single-cluster/install-all-in-one.yaml
+}
+
+function patch_quickstart_images() {
+  local quickstart_digest
+  local adapter_quickstart_digest
+
+  quickstart_digest=$(fetch_digest syntasso/kratix-platform-quickstart)
+  validate_digest "KRATIX_QUICKSTART_MANIFEST_DIGEST" "$quickstart_digest"
+
+  adapter_quickstart_digest=$(fetch_digest syntasso/kratix-platform-pipeline-adapter-quickstart)
+  validate_digest "KRATIX_QUICKSTART_PIPELINE_ADAPTER_MANIFEST_DIGEST" "$adapter_quickstart_digest"
 
-make distribution
+  export QUICKSTART_TAG="syntasso.docker.scarf.sh/syntasso/kratix-platform-quickstart@${quickstart_digest}"
+  export QUICKSTART_PIPELINE_ADAPTER_TAG="syntasso.docker.scarf.sh/syntasso/kratix-platform-pipeline-adapter-quickstart@${adapter_quickstart_digest}"
 
-mkdir -p distribution/single-cluster
+  yq eval '
+    select(.kind == "Deployment" and .metadata.name == "kratix-platform-controller-manager")
+    .spec.template.spec.containers[0].image = strenv(QUICKSTART_TAG)
+    // select(.kind != "Deployment" or .metadata.name != "kratix-platform-controller-manager")
+  ' distribution/single-cluster/install-all-in-one.yaml > tmp.yaml && mv tmp.yaml distribution/single-cluster/install-all-in-one.yaml
 
-cat distribution/kratix.yaml <(echo "---") \
-hack/destination/gitops-tk-install.yaml <(echo "---") \
-hack/platform/minio-install.yaml > distribution/single-cluster/install-all-in-one.yaml
+  yq eval '
+    select(.kind == "ConfigMap" and .metadata.name == "kratix-platform-pipeline-adapter-config")
+    .data.PIPELINE_ADAPTER_IMG = strenv(QUICKSTART_PIPELINE_ADAPTER_TAG)
+    // select(.kind != "ConfigMap" or .metadata.name != "kratix-platform-pipeline-adapter-config")
+  ' distribution/single-cluster/install-all-in-one.yaml > tmp.yaml && mv tmp.yaml distribution/single-cluster/install-all-in-one.yaml
+}
 
-cat config/samples/platform_v1alpha1_worker.yaml <(echo "---") \
-config/samples/platform_v1alpha1_bucketstatestore.yaml <(echo "---") \
-hack/destination/gitops-tk-resources-single-cluster.yaml > distribution/single-cluster/config-all-in-one.yaml
+generate_distribution_manifests
+patch_quickstart_images
```

Comment:
not sure i'm a fan of the generate and patch approach; wouldn't be easier/simpler to do something like:

```bash
generate_quickstart_manifests
generate_distribution_manifests
```

where "generate_quickstart_manifests" runs the `make distribution` setting the right `PIPELINE_ADAPTER_IMG` and `IMG_TAG`?